include/bluetooth/services/cgms.h

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef BT_CGMS_H_
+#define BT_CGMS_H_
+
+#include <zephyr/types.h>
+#include <zephyr/bluetooth/conn.h>
+
+/**@file
+ * @defgroup bt_cgms Continuous Glucose Monitoring Service API
+ * @{
+ * @brief API for the Continuous Glucose Monitoring Service (CGMS).
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief CGMS Feature characteristic Type field */
+enum bt_cgms_feature_type {
+	/** Capillary Whole blood */
+	BT_CGMS_FEAT_TYPE_CAP_BLOOD = 0x01,
+	/** Capillary Plasma */
+	BT_CGMS_FEAT_TYPE_CAP_PLASMA = 0x02,
+	/** Venous Whole blood */
+	BT_CGMS_FEAT_TYPE_VEN_BLOOD = 0x03,
+	/** Venous Plasma */
+	BT_CGMS_FEAT_TYPE_VEN_PLASMA = 0x04,
+	/** Arterial Whole blood */
+	BT_CGMS_FEAT_TYPE_ART_BLOOD = 0x05,
+	/** Arterial Plasma */
+	BT_CGMS_FEAT_TYPE_ART_PLASMA = 0x06,
+	/** Arterial Plasma */
+	BT_CGMS_FEAT_TYPE_UNDET_BLOOD = 0x07,
+	/** Arterial Plasma */
+	BT_CGMS_FEAT_TYPE_UNDET_PLASMA = 0x08,
+	/** Interstitial Fluid (ISF) */
+	BT_CGMS_FEAT_TYPE_FLUID = 0x09,
+	/** Control Solution */
+	BT_CGMS_FEAT_TYPE_CONTROL = 0x0A,
+};
+
+/** @brief CGMS Feature characteristic Sample Location field */
+enum bt_cgms_feature_sample_location {
```

Comment:
Enum type name should be a prefix for enum item names:

```suggestion
enum bt_cgms_feat_loc {
```